src/test/kotlin/com/zepben/evolve/services/network/testdata/HvEquipmentBelowLvFeederHeadNetwork.kt

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.services.network.testdata
+
+import com.zepben.evolve.cim.iec61970.base.core.BaseVoltage
+import com.zepben.evolve.services.network.NetworkService
+import com.zepben.evolve.testing.TestNetworkBuilder
+
+object HvEquipmentBelowLvFeederHeadNetwork {
+
+    //
+    // - or |: LV line
+    // = or #: HV line
+    //
+    //      c1     c3     c7         c9     c10
+    // b0 ------+------+------ tx8 ======+------
+    //          #      |
+    //       c2 #   c4 |
+    //          #      |
+    //                tx5
+    //                 #
+    //              c6 #
+    //                 #
+    //
+    // lvf11 head terminal is b0-t2
+    //
```

Comment:
suggestion: This network looks like it is a copy of one built for older tests that includes more in them than is required for each test, and tries to cover lots of tests with a single network. Consider reducing the size of each tests network to only what is needed to carry out that test, even if that means more copies of similar networks.

e.g. If you are testing that a transforer is included in the LV network when moving from LV to HV you only need a network with LV into a TX into HV.

The aobve network looks like it also has network for testing that the trace does not continue through HV back onto LV, which should be covered by testing the trace algorithm, so no need to do this again in the LV feeder assigner.